pkg/descheduler/strategies/duplicates_test.go

```diff
@@ -62,24 +79,30 @@ func TestFindDuplicatePods(t *testing.T) {
 			},
 		},
 	}
+
 	// A Mirror Pod.
 	p6.Annotations = test.GetMirrorPodAnnotation()
+
 	// A Critical Pod.
-	p7.Namespace = "kube-system"
 	p7.Annotations = test.GetCriticalPodAnnotation()
-	expectedEvictedPodCount := 2
+
+	// Setup the fake client.
 	fakeClient := &fake.Clientset{}
 	fakeClient.Fake.AddReactor("list", "pods", func(action core.Action) (bool, runtime.Object, error) {
-		return true, &v1.PodList{Items: []v1.Pod{*p1, *p2, *p3, *p4, *p5, *p6, *p7, *p8, *p9}}, nil
+		return true, &v1.PodList{Items: []v1.Pod{*p1, *p2, *p3, *p4, *p5, *p6, *p7, *p8, *p9, *p10}}, nil
 	})
 	fakeClient.Fake.AddReactor("get", "nodes", func(action core.Action) (bool, runtime.Object, error) {
 		return true, node, nil
 	})
+
+	expectedEvictedPodCount := 4
 	npe := nodePodEvictedCount{}
 	npe[node] = 0
-	podsEvicted := deleteDuplicatePods(fakeClient, "v1", []*v1.Node{node}, false, npe, 2, false)
```

Comment:
Since `maxPodsEvicted` was 2, the test would pretty much always succeed due to `expectedEvictedPodCount` being the same value. I've upped `maxPodsEvicted` to 10 now, the amount of Pods currently created.